h2o-core/src/main/java/water/rapids/ast/prims/internal/AstRunTool.java

```diff
@@ -0,0 +1,44 @@
+package water.rapids.ast.prims.internal;
+
+import water.rapids.Env;
+import water.rapids.ast.AstPrimitive;
+import water.rapids.ast.AstRoot;
+import water.rapids.vals.ValStr;
+
+import java.lang.reflect.Method;
+
+public class AstRunTool extends AstPrimitive<AstRunTool> {
+
+    private static final String TOOLS_PACKAGE = "water.tools.";
+
+    @Override
+    public String[] args() {
+        return new String[]{"tool_class", "tool_parameters"};
+    }
+
+    @Override
+    public int nargs() {
+        return 1 + 2;
```

Comment:
run_tool =1 ;tool_class =1; tool_parameters = 1;
return run_tool + tool_class + tool_parameters;
;-)

or ?
run_tool =1;
return run_tool + args().length

Reply:
drop in a bucket :) it is just how the rapids are done, IMHO keeping it consistent is better, wdyt?